waffle_configs/bedrock-prod.env

```diff
@@ -25,7 +25,7 @@ SWITCH_EXPERIMENT_FIREFOX_HOME_PRE_DOWNLOAD=off
 SWITCH_EXPERIMENT_FIREFOX_HOME_PRE_DOWNLOAD_FR=on
 SWITCH_EXPERIMENT_FIREFOX_HOME_PRE_DOWNLOAD_DE=on
 SWITCH_EXPERIMENT_FIREFOX_HOME_PRE_DOWNLOAD_LOCALES=on
-SWITCH_EXPERIMENT_FIREFOX_NEW_WIN10_TASKBAR_FUNNELCAKE=off
+SWITCH_EXPERIMENT_FIREFOX_NEW_WIN10_TASKBAR_FUNNELCAKE=on
```

Comment:
Rather than set this on immediately and then test live in production, I'd suggest we do this for stage first and then prod in a separate PR?

Reply:
That's what #182 is for :)

Reply:
Ah, missed that there we're 2 PR's open 👍